python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -212,14 +229,15 @@ async def begin_start(self, data):
         data : `DataType`
             Command data
         """
-
+        self.log.debug('At beginning of begin_start')
         if self.pneumatics_summary_state is None:
             try:
                 await self.pneumatics.evt_summaryState.next(flush=False,
                                                             timeout=self.fast_timeout)
             except asyncio.TimeoutError:
                 self.log.warning("Could not get summary state from ATPneumatics.")
             # Trick to get last value, until aget is available.
+            # FIXME: Can we remove this now and use aget above?
```

Comment:
yes... do you want to go ahead and fix it here? :)

Reply:
Will do... wanted confirmation first...